utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -67,7 +67,7 @@ int main(int argc, char **argv)
     bool additionalParamCase = (testCase == 8 || testCase == 21 || testCase == 23 || testCase == 24 || testCase == 49 || testCase == 79);
     bool kernelSizeCase = (testCase == 49);
     bool dualInputCase = (testCase == 2 || testCase == 30 || testCase == 33 || testCase == 61 || testCase == 63 || testCase == 65 || testCase == 68);
-    bool randomOutputCase = (testCase == 6 || testCase == 8 || testCase == 84);
```

Comment:
I have commented many times before. We should not use numbers for test cases. It is better to use ENUMs here